testsuite/MDAnalysisTests/topology/test_gro.py

```diff
@@ -90,38 +84,36 @@ class TestGroResidWrapping(object):
     lengths = [19, 24, 19, 19, 19, 19, 7]
     parser = mda.topology.GROParser.GROParser
 
+    @pytest.mark.parametrize('parser, resids', (
+        (GRO_residwrap, [1, 99999, 100000, 100001, 199999, 200000, 200001]),
+        (GRO_residwrap_0base, [0, 99999, 100000, 100001, 199999, 200000,
+                               200001])
 
-    def test_wrapping_resids(self):
-        parser = mda.topology.GROParser.GROParser
-        with self.parser(GRO_residwrap) as p:
+    ))
+    def test_wrapping_resids(self, parser, resids):
+        with self.parser(parser) as p:
             top = p.parse()
 
-        resids = [1, 99999, 100000, 100001, 199999, 200000, 200001]
-
-        assert_(top.tt.size == (126, 7, 1))
+        assert top.tt.size == (126, 7, 1)
         for i, (r, n, l) in enumerate(zip(resids, self.names, self.lengths)):
-            assert_(top.resids.values[i] == r)
-            assert_(top.resnames.values[i] == n)
-            assert_(len(top.tt.residues2atoms_1d([i])) == l)
+            assert top.resids.values[i] == r
+            assert top.resnames.values[i] == n
+            assert len(top.tt.residues2atoms_1d([i])) == l
 
-    def test_wrapping_resids_0base(self):
-        with self.parser(GRO_residwrap_0base) as p:
-            top = p.parse()
-
-        resids = [0, 99999, 100000, 100001, 199999, 200000, 200001]
 
-        assert_(top.tt.size == (126, 7, 1))
-        for i, (r, n, l) in enumerate(zip(resids, self.names, self.lengths)):
-            assert_(top.resids.values[i] == r)
-            assert_(top.resnames.values[i] == n)
-            assert_(len(top.tt.residues2atoms_1d([i])) == l)
+RESIDS = [9, 9]
+RESNAMES = ['GLN', 'POPC']
 
-def test_sameresid_diffresname():
+@pytest.mark.parametrize(
```

Comment:
this doesn't really need to be parametrized, it all tests the same thing, rather than different cases of something